doc/repro.rst

```diff
@@ -324,6 +330,9 @@ MESSAGEix-GLOBIOM 1.1
 
    .. todo:: Expand with a list of cases in which this model name has been used.
 
+MESSAGEix-GLOBIOM 1.2
+   This version was used in <https://doi.org/10.1038/s41560-021-00904-8>, and is similar to MESSAGEix-GLOBIOM 1.1, but incorporates COVID effects and has annual timesteps from 2020 to 2025.
```

Comment:
Please add additional citations to doc/main.bib and cite; see the example of the rest of the docs.